src/main/java/com/mojang/brigadier/StringReader.java

```diff
@@ -177,6 +247,13 @@ public String readUnquotedString() {
         return string.substring(start, cursor);
     }
 
+    /**
+     * Returns a quoted string.
```

Comment:
Sounds good :)
Unfortunately the fields are private for some reason, so they would not appear in the generated javadoc (It would just place an empty String there), so I will literally duplicate them and hope I am not forcing the API to follow something it does not intend to keep. But quotes are hopefully unlikely to change ;)